### PR TITLE
Issue#2437

### PR DESCRIPTION
**Pull request by ArthurNiedial**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [x] No need to
- [ ] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/extensions/WebUtils.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import com.codeborne.selenide.SelenideElement;
+
+public class WebUtils {
+
+    public static boolean isVisible(SelenideElement element) {
+        boolean isVisible = false;
+        try {
+            isVisible = element.is(Condition.visible);
+        } catch (Throwable e) {
+            e.printStackTrace();
```
Comment (Haarolean) [inline kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/extensions/WebUtils.java:13]: use slf4j for this please

File: kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/extensions/WebUtils.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.SelenideElement;
+
+public class WebUtils {
+
+    public static boolean isVisible(SelenideElement element) {
+        boolean isVisible = false;
+        try {
+            isVisible = element.is(Condition.visible);
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/extensions/WebUtils.java:11]: let's use smth like this here:
element.shouldBe(Condition.visible);
isVisible = true;

File: kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/connector/ConnectorsList.java [hunk 3]
```diff
@@ -50,11 +51,11 @@ public ConnectorsList isNotVisible(String connectorName) {
     }
 
     @Step("Verify that connector {connectorName} is visible in the list")
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/connector/ConnectorsList.java:53]: step description is not actual

File: kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/schema/SchemaView.java [hunk 4]
```diff
@@ -20,9 +20,8 @@ public SchemaView waitUntilScreenReady() {
     }
 
     @Step
-    public SchemaView isCompatibility(CompatibilityLevel.CompatibilityEnum compatibility){
-        $x("//div//p[.='" + compatibility.getValue() + "']").shouldBe(Condition.visible);
-        return this;
+    public boolean isCompatibility(CompatibilityLevel.CompatibilityEnum compatibility){
+        return    $x("//div//p[.='" + compatibility.getValue() + "']").is(Condition.visible);
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/schema/SchemaView.java:24]: 1. extra spaces
2. really don't understand what does this method do - isCompatibility what?? Equals? Visible? imho (i didn't dive into context) we need to assert Compatibility value, so method should be getCompatibility() and return it's value that will be handlet in test using Assert.Equals.. if i'm not right let's discuss

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java [hunk 5]
```diff
@@ -53,8 +54,9 @@ void createSchemaAvro() {
                 .waitUntilScreenReady();
         pages.mainPage
                 .goToSideMenu(CLUSTER_NAME, MainPage.SideMenuOptions.SCHEMA_REGISTRY);
-        pages.schemaRegistry.isSchemaVisible(schemaAvro.getName());
+        Assertions.assertTrue(pages.schemaRegistry.isSchemaVisible(schemaAvro.getName()),"isSchemaVisible()");
         SCHEMA_LIST.add(schemaAvro);
+
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java:59]: extra spaces

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java [hunk 6]
```diff
@@ -51,22 +53,21 @@ public void createTopic() {
                 .sendData()
                 .waitUntilScreenReady();
         pages.open()
-                .goToSideMenu(CLUSTER_NAME, MainPage.SideMenuOptions.TOPICS)
-                .topicIsVisible(topicToCreate.getName());
+                .goToSideMenu(CLUSTER_NAME, MainPage.SideMenuOptions.TOPICS);
+        Assertions.assertTrue(pages.topicsList.isTopicVisible(topicToCreate.getName()));
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java:57]: we need to leave assertion message in every place we use assertions

**Review by VladSenyuta — CHANGES_REQUESTED** [review]

File: kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/extensions/WebUtils.java (hunk 1 above)
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/extensions/WebUtils.java]: Fixed

File: kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/extensions/WebUtils.java (hunk 2 above)
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/extensions/WebUtils.java]: Fixed

File: kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/connector/ConnectorsList.java (hunk 3 above)
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/connector/ConnectorsList.java:53]: Fixed

File: kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/schema/SchemaView.java (hunk 4 above)
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/schema/SchemaView.java]: Fixed

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java (hunk 5 above)
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java]: Fixed

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java (hunk 6 above)
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java]: Fixed

File: kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/connector/ConnectorsList.java (hunk 3 above)
Comment (VladSenyuta) [reply kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/connector/ConnectorsList.java:53]: can't see anyy changes here. where did you fix it??

Comment (VladSenyuta) [reply kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/connector/ConnectorsList.java:53]: can't see any changes here. where did you fix it??

Comment (VladSenyuta) [reply kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/connector/ConnectorsList.java:53]: can't see any changes here. where did you fix it??

Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/connector/ConnectorsList.java:53]: ??

**Review by VladSenyuta — CHANGES_REQUESTED** [review]

**Review by VladSenyuta — APPROVED** [review]

**Pull request merged**